### ISSUE TITLE
Trakt.tv authentication required in 0.9.27 and 0.9.28

### ISSUE THREAD

**Issue by MichaelAndoid** [scakemyer/plugin.video.quasar#383]
Hi today i update to version 0.9.2.8 and now i cant navigate from Quasar always is asking for this:
```
**Autorize Quasar**
Visit https://trakt/.tv/activate and enter your code: xxxxxxxxxx 
```

Before this i never need for the register in the trakt.tv, this is a bug or now i have to do this every-time i need to use Quasar 0.9.2.8?

**Comment by scakemyer** [comment]
It's a bug, it was supposed to only be required when accessing/adding/removing from watchlist and collection. Pushing another hotfix release within the next hour for this one.

**Comment by scakemyer** [comment]
Fixed in v0.9.29

**Comment by tomtomclub** [comment]
Lol  wahh now I have trak acoud  werk nice men
Thanks  scakemyer
